code/game/objects/structures/transit_tubes.dm

```diff
@@ -125,17 +125,44 @@ obj/structure/transit_tube_pod/ex_act(severity)
 	..()
 
 
-/obj/structure/transit_tube/station/attack_hand(mob/user as mob)
+/obj/structure/transit_tube/station/attack_hand(mob/user)
 	if(!pod_moving)
 		for(var/obj/structure/transit_tube_pod/pod in loc)
 			if(!pod.moving && pod.dir in directions())
 				if(open)
+					if(!user.lying && user.loc != pod)
```

Comment:
I don't think these are likely to change between one for-loop iteration and the other, could place it outside.